lib/video/screenRecording/firefox/firefoxWindowRecorder.js

```diff
@@ -32,7 +32,7 @@ function findRecordingDirectory(baseDir) {
 
   fs.readdirSync(baseDir).forEach(file => {
     if (file.startsWith('windowrecording-')) {
-      let fullPath = path.join(baseDir,file);
+      let fullPath = path.join(baseDir, file);
       let mtime = fs.statSync(fullPath).mtime;
```

Comment:
Consider making this one a `.reduce()` as well, since we're not really finding "closest" anything, we're finding "latest".